specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -7688,130 +8132,492 @@
           "type": "string",
+          "readOnly": true,
           "format": "date-time",
-          "description": "Time to which the account has to be restored (ISO-8601 format)."
-        },
-        "databasesToRestore": {
-          "type": "array",
-          "description": "List of specific databases to restore.",
-          "items": {
-            "$ref": "#/definitions/DatabaseRestoreResource"
-          }
+          "description": "The time at which the regional restorable database account has been deleted (ISO-8601 format)."
         }
       }
     },
-    "DatabaseRestoreResource": {
-      "type": "object",
-      "description": "Specific Databases to restore.",
+    "RestorableSqlDatabasesListResult": {
       "properties": {
-        "databaseName": {
-          "type": "string",
-          "description": "The name of the database to restore."
-        },
-        "collectionNames": {
+        "value": {
+          "readOnly": true,
           "type": "array",
-          "description": "The names of the collections to restore.",
           "items": {
-            "$ref": "#/definitions/CollectionName"
-          }
+            "$ref": "#/definitions/RestorableSqlDatabaseGetResult"
+          },
+          "description": "List of restorable SQL databases and their properties."
         }
-      }
+      },
+      "description": "The List operation response, that contains the restorable SQL databases and their properties."
     },
-    "CollectionName": {
-      "type": "string",
-      "description": "The name of the collection."
+    "RestorableSqlDatabaseGetResult": {
+      "description": "An Azure Cosmos DB restorable SQL database",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a restorable SQL database.",
+          "$ref": "#/definitions/RestorableSqlDatabaseProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ARMResourceProperties"
+        }
+      ]
     },
-    "BackupPolicy": {
+    "RestorableSqlDatabaseProperties": {
+      "description": "The properties of an Azure Cosmos DB restorable SQL database",
       "type": "object",
-      "description": "The object representing the policy for taking backups on an account.",
-      "discriminator": "type",
       "properties": {
-        "type": {
-          "type": "string",
-          "enum": [
-            "Periodic",
-            "Continuous"
-          ],
-          "description": "Describes the mode of backups.",
-          "x-ms-enum": {
-            "modelAsString": true,
-            "name": "BackupType"
+        "resource": {
+          "properties": {
+            "_rid": {
+              "type": "string",
+              "readOnly": true,
+              "description": "A system generated property. A unique identifier."
+            },
+            "operationType": {
```

Comment:
OperationType will have values from a fixed set ? If so, can we make it a Enum ?

Reply:
sure. made it enum